metacat-main/src/test/groovy/com/netflix/metacat/main/search/BaseEsSpec.groovy

```diff
@@ -24,13 +24,15 @@ import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsReques
 import org.elasticsearch.client.Client
 import org.elasticsearch.common.settings.ImmutableSettings
 import org.elasticsearch.common.settings.Settings
+import org.junit.Ignore
 import spock.lang.Shared
 import spock.lang.Specification
 
 /**
  * Created by amajumdar on 8/17/15.
  */
-class BaseEsSpec extends Specification {
+@Ignore
```

Comment:
@ajoymajumdar note this is the same change you made in #116 I had to put it in here as I couldn't get the build to pass this step due to the multiple runs issue you were talking about.